.github/workflows/sdist.yml

```diff
@@ -38,17 +38,9 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
 
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip setuptools wheel
-
-        # GH 39416
-        pip install numpy
-
     - name: Build pandas sdist
       run: |
-        pip list
```

Comment:
i think we usually want this line so that we can see what versions are installed when debugging CI failures

Reply:
This is not the build environment. No version listed here would have anything to do with what you have inside the PEP 517 build environment. If you use pipx, as below, then even the version of build is not listed and the version of Python is unrelated.